monthly_data_report.R

```diff
@@ -1,28 +1,37 @@
-#                                MANDATED_REPORTER = col_character(),
-#                                #  REPORTER_PERSON_ID = col_character(),
-#                                address_type = col_character(),
-#                                address = col_character(),
-#                                bad_address = col_logical(),
-#                                PO = col_logical(),
-#                                lat = col_double(),
-#                                lon = col_double(),
-#                                score = col_double(),
-#                                precision = col_character(),
-#                                precise_geocode = col_logical(),
-#                                census_block_group_id_2010 = col_character(),
-#                                census_tract_id_2010 = col_character(),
-#                                BIRTH_DATE = col_date(format = "%m/%d/%Y"),
+#                                                 SCREENING_DECISION = col_character(),
+#                                                 DECISION_DATE = col_date(format = "%m/%d/%y"),#format = "%m/%d/%Y"),
+#                                                 BIRTH_DATE = col_date(format = "%m/%d/%y"),#format = "%m/%d/%Y"),
+#                                                 PERSON_ID = col_character(),
+#                                                 RACE = col_character(),
+#                                                 ADDRESS_START = col_date(format = "%m/%d/%y"),
+#                                                 REPORTER_PERSON_ID = col_character(),
+#                                                 MANDATED_REPORTER = col_character(),
+#                                                 address_type = col_character(),
+#                                                 address = col_character(),
+#                                                 bad_address = col_logical(),
+#                                                 PO = col_logical(),
+#                                                 matched_street = col_character(),
+#                                                 matched_zip = col_character(),
+#                                                 matched_city = col_character(),
+#                                                 matched_state = col_character(),
+#                                                 lat = col_double(),
+#                                                 lon = col_double(),
+#                                                 score = col_double(),
+#                                                 precision = col_character(),
+#                                                 geocode_result = col_character(),
+#                                                 census_block_group_id_2010 = col_character(),
+#                                                 census_tract_id_2010 = col_character()
 #               ))
 
+library(tidyr)
+library(readr)
+library(dplyr)
```

Comment:
could add `warn.conflicts = FALSE` to the library calls here so that the masking messages don't show up for the DeGAUSS user when they run the container.